server/index.js

```diff
@@ -399,72 +399,435 @@ function buildAppHTML (myuserid, user, arrayOfMatches) {
+                    
+                    <div class="card profile-card">
+                        <object class="card-img-top" data="${user.profile_picture}" type="image/png">
+                            <img class="card-img-top" src="${defaultPhoto}" alt="Card image cap">
+                        </object>
+                        <div class="card-body">
+                            <div class="d-flex justify-content-between" id="card-header">
+                                <div class="card-title" id="profile-name">${user.f_name}</div>
+                                <div class="card-title" id="profile-age">${user.age}</div>
+                            </div>
+                        
+                        <form method="post" action="/Profile">
+                        <div class="input-group">
+                            <div class="input-group-prepend">
+                                <span class="input-group-text">About Me:</span>
+                            </div>
+                            <textarea class="form-control" aria-label="With textarea" name="bio" >${user.bio}</textarea>
+                            <input type="submit" class="btn-block btn-danger" data-inline="true" value="Save">
+                        </div>
+                        </form>
+
+                        </div>
+                    </div>
+                    </div>
+
+                    </div>
+                
+                
+                </div>
+            </div>
+        </div>
+
+    </div>          <label style="color:#000;" class="mr-sm-2" for="inlineFormCustomSelect">I am looking for:</label>
+                   <body style="background-color:#686868">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="sidenav" style="background-color:#f9f3f2;">
+                    <form class="form-signout" id='logout-form' action="/logout" method="post">  
+                        <button class="btn-md btn-danger btn-block" type="submit">Sign out</button>
+                    </form>
+                    <h4 style="text-align:center; color: #fff; font-weight: 800; background-color:#ff5050; padding:25px;"><i class="fas fa-fire"></i>   Timber</h4>
+                    <form>
+                        <input type="button" class="btn-danger" value="Back" onclick="history.back()">
+                    </form>
+
+                    <p style="text-align:center; background-color:#ff5050;">Preferences</p>
+                    
+                    <form method="post" action="/Preferences"><br />
+                
+                        <div class="col-auto my-1">
+                            <label style="color:#000;" class="mr-sm-2" for="inlineFormCustomSelect">I am looking for:</label>
+                            <select class="custom-select mr-sm-2" id="inlineFormCustomSelect" name="pref_gender">
+                            <option selected>Choose one</option>
```

Comment:
@gwin333 this was fixed on master. selected option should have the value of the user's pref_gender